notebooks_tsqr/NightLog.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.          f' from {min_day_obs} to {date}. ')
</pre><p>I think for our initial investigating we were allowing many days and different days, etc, but I think when possible/relevant we should shift the focus more towards 'what happened last night' and allow that to be the default assumption of what users would be interested in. yagni</p><p>This comment applies to record_limit, number_of_days, and this message</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_logging_and_reporting/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Without number_of_days greater than one, users may find it difficult to see if results look good.  If they happen to have a day with no results (happens on usdf-dev frequently) the only remedy would be to try another day and see if that works.  Repeat a few times and its frustrating.   In this case to pick one day, we have to have a min and max day so we already have to have a number of days concept that finds its way into code.  After everyone agrees on the format of displays, we can remove the number_of_days parameter from sidecar yaml file so user will always get the default of 1.